packages/calypso-e2e/src/lib/pages/user-signup-page.ts

```diff
@@ -35,6 +36,24 @@ export class UserSignupPage {
 		this.page = page;
 	}
 
+	/**
+	 * Navigates to the /log-in endpoint.
+	 */
+	async visitBlueSignup( locale = '' ): Promise< void > {
+		const targetUrl = locale ? `start/premium/${ locale }` : 'start/premium';
+		console.log( targetUrl );
+		await this.page.goto( getCalypsoURL( targetUrl ), { waitUntil: 'networkidle' } );
+	}
+
+	/**
+	 * Navigates to the /log-in endpoint.
+	 */
+	async visitWhiteSignup( locale = '' ): Promise< void > {
+		const targetUrl = locale ? `start/${ locale }` : 'start';
+		console.log( targetUrl );
+		await this.page.goto( getCalypsoURL( targetUrl ), { waitUntil: 'networkidle' } );
+	}
+
```

Comment:
With changes proposed in the `LoginPage` POM, changes here should no longer be necessary.